html/changelogs/AutoChangeLog-pr-30416.yml

```diff
@@ -0,0 +1,4 @@
+author: "Xhuis"
```

Comment:
Remove this file from your PR

Reply:
It'd help if you told me how to, since I am new to using this Git Tortoise thing.